app/src/features/apiClient/components/RouteElement.tsx

```
@@ -1,20 +1,244 @@
+  const last = useRef<ScopedVariables>(null)
+
+  // const s = makeScopeChain('9130594d-2515-47d6-9371-18febb62a8a2', vh);
+  const store = useWorkspaceViewStore();
+  const scopedVariables = useScopedVariables('9130594d-2515-47d6-9371-18febb62a8a2');
+
+  useEffect(() => {
+    if(!last.current && scopedVariables) {
+      last.current = scopedVariables;
+    }
+    // const areSame = s(store.getState()) === s(store.getState());
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)